app/application.py

```diff
@@ -74,7 +74,7 @@ def before_request():
         log_request()
         res = authenticate(request.headers)
         if res != {'status': "ok"}:
-            logger.debug("Failed to authenticate user")
+            logger.debug('Failed to authenticate user')
```

Comment:
The user uuid gets logged during the authenticate call. But we will include the status in the log message.